core/src/main/resources/templates/email/messages.properties

```diff
@@ -5,9 +5,16 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 ###############################################################################
-buddy-invitation-subject=Yona buddy invitation from {0}
-buddy-invitation-body.salutation=Hi {0} {1}, become my buddy for Yona!
-buddy-invitation-body.body.p1=As we have talked about, I'd like you to become my buddy in the Yona system. If you didn't install the Yona app yet, please start with doing that. Here are the links:...
-buddy-invitation-body.body.p2=Next, open the below link by tapping on it and selecting Yona as the app to open it with.
-buddy-invitation-body.body.warning=Important: Validate that {0} {1} has really invited you. The mobile number registered with Yona for this user is {2}.
-buddy-invitation-body.body.greeting=Best regards,<br/>{0} {1} | {2}
\ No newline at end of file
+buddy-invitation-subject=Become my friend on Yona!
+buddy-invitation-body.header=Become friends on Yona!
+buddy-invitation-body.p1={0} has invited you to become friends on Yona. Yona enables you to make sure together the Internet remains fun. Follow the steps below and see which goals have been set by {0}. After that, set your own goals.
```

Comment:
Propose to change
``Yona enables you to make sure together the Internet remains fun.``
into
``With Yona, you together ensure Internet stays fun.``